### PR TITLE
Fix ZombieTServer metrics in for LateLastContactIT failure

### PR DESCRIPTION
**Pull request by EdColeman**
Prevent NPE with DistributionSummary in ThriftMetrics if metrics are not initialized.  

   - Fixes failures in LateLastContactIT by initializing the metrics
   - Adds a general guard by providing a default, no-op distribution summary.
 
The initialization of the metrics the ZombieTServber was already made in main as part of #4500.

### CODE REVIEW

File: server/tserver/src/main/java/org/apache/accumulo/tserver/metrics/TabletServerUpdateMetrics.java [hunk 1]
```diff
@@ -35,7 +36,7 @@ public class TabletServerUpdateMetrics implements MetricsProducer {
   private Timer commitPrepStat;
   private Timer walogWriteTimeStat;
   private Timer commitTimeStat;
-  private DistributionSummary mutationArraySizeStat;
+  private DistributionSummary mutationArraySizeStat = new NoOpDistributionSummary();;
```
Comment (kevinrr888) [inline server/tserver/src/main/java/org/apache/accumulo/tserver/metrics/TabletServerUpdateMetrics.java:39]:
```suggestion
  private DistributionSummary mutationArraySizeStat = new NoOpDistributionSummary();
```
Unnecessary semi colon

File: server/base/src/test/java/org/apache/accumulo/server/metrics/ThriftMetricsTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  void registerMetrics() {
+    MeterRegistry registry = new SimpleMeterRegistry();
+    ThriftMetrics tm = new ThriftMetrics();
+    tm.registerMetrics(registry);
+    tm.addExecute(1000);
+    tm.addIdle(1000);
+
+    registry.forEachMeter(m -> {
+      LOG.trace("meter: {}", m.getId());
+      assertTrue(m instanceof DistributionSummary);
```
[the remark this answers is not in the document]
Comment (EdColeman) [reply server/base/src/test/java/org/apache/accumulo/server/metrics/ThriftMetricsTest.java]: fixed in 98e22de70e

File: server/tserver/src/main/java/org/apache/accumulo/tserver/metrics/TabletServerUpdateMetrics.java (hunk 1 above)
Comment (EdColeman) [reply server/tserver/src/main/java/org/apache/accumulo/tserver/metrics/TabletServerUpdateMetrics.java]: removed in 98e22de70e

File: server/base/src/main/java/org/apache/accumulo/server/metrics/NoOpDistributionSummary.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+ *      distSum = DistributionSummary.builder("metric name") .description("...").register(registry);
+ *      ...
+ *   }
+ * </pre>
+ *
+ * Until the registration is called, the instance variable is null. If code then tries to update the
+ * metric, a NPE is thrown. Using this class to provide a default value prevents this from occurring
+ * and on registration, it is replaced with a valid instance.
+ */
+public class NoOpDistributionSummary implements DistributionSummary {
```
[the remark this answers is not in the document]
Comment (EdColeman) [reply server/base/src/main/java/org/apache/accumulo/server/metrics/NoOpDistributionSummary.java:49]: Adding logging (at debug) in 8aaf8776da

**Pull request merged**